api/pkg/transformer/executor/predictor.go

```diff
@@ -0,0 +1,47 @@
+package executor
```

Comment:
Shall we call the package `simulator`? Or `executor` is more appropriate?

Reply:
the purpose not only for simulation but also execute transformer in general, so in the future suppose turing want to call standard transformer it can orchestrate the executor from turing itself by supplying modelPredictor implementation (HTTP or mock)